stests/core/cache/ops_infra.py

```diff
@@ -65,6 +65,19 @@ def get_network(network_id: NetworkIdentifier) -> Network:
     ]
 
 
+def get_network_by_ctx(ctx: ExecutionContext) -> Network:
+    """Decaches domain object: Network.
+    
+    :param ctx: Execution context information.
+
+    :returns: A registered network.
+
+    """
+    network_id = factory.create_network_id(ctx.network)
```

Comment:
It's a little hard to read the codebase because you reuse terms [orchestration, factory, .. ] in different folders and so in the codebase, with the variables named like `factory` I have to go verify which factory it is referring to.

For example:
- `stests/orchestration/factory.py`
- `stests/core/utils/factory.py`

Then orchestration is in both
- `stests/core/orchestration`
- `stests/orchestration`

I'll tell you where this'll be problematic: `import *`

If I have another `factory` in a module at some point later, it'll clash. 

First, we should change `import *` and then either rename the modules or the variables referring to them.

Reply:
I understand that there is a source of confusion here.  In respect of the factories I was thinking of doing some work - like make factory a top-level package with sub-modules.  

With the import * I am disciplined in it's use - i.e. I rarely use it except for in __init__ files or in a package like domain where I deliberately want to import the declared types.  But point taken.

In relation to the orchestration code there is indeed a degree of debate to be had in respect of stests/core/orchestration & stests/orchestration.  I will review the code and perhaps make some changes to simplify understanding.